pingen/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Author: Guewen Baconnier
+# Copyright 2012-2017 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "pingen.com integration",
+    "version": "16.0.1.0.0",
+    "author": "Camptocamp,Odoo Community Association (OCA)",
+    "maintainer": "Camptocamp, Akim Juillerat, Anna Janiszewska",
```

Comment:
```suggestion
    "maintainers": ["ajaniszewska-dev", "grindtildeath"],
```
cf https://odoo-community.org/page/maintainer-role